scripts/maint/post-merge.git-hook

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+git_toplevel=$(git rev-parse --show-toplevel)
+
+check_for_diffs() {
+        installed="$git_toplevel/.git/hooks/$1"
+        latest="$git_toplevel/scripts/maint/$1.git-hook"
+
+        if [ -e "$installed" ]
+        then
+               if ! cmp "$installed" "$latest" >/dev/null 2>&1
+               then
+                        echo "ATTENTION: $1 hook has changed:"
+                        echo "==============================="
+                        diff "$installed" "$latest"
```

Comment:
If we keep this line, it should probably be `diff -u`

Reply:
Did so in 48a3384df80d3833e99adf246971bdb9a491521b.